weave-js/src/components/PagePanelComponents/Home/Browse3/typeViews/Audio/AudioPlayer.tsx

```diff
@@ -40,10 +53,11 @@ export const AudioPlayer: FC<{
   }
 
   const downloadFile = () => {
+    if (!audioUrl || !audioFileName) return;
     const a = document.createElement('a');
     a.href = audioUrl;
     const date = new Date().toISOString().split('T')[0];
-    a.download = `${project}_${date}_audio.wav`;
+    a.download = `${project}_${date}_${entity}_${audioFileName}`;
```

Comment:
confirming audioFileName includes the media type right?

Reply:
Yup, audioFileName includes the extension 
![image](https://github.com/user-attachments/assets/6ba16271-c8a7-4592-a43a-937a4ee8a9fb)